apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -85,10 +89,91 @@ public partial class BigtableServiceApiSettings
                 retryFilter: IdempotentRetryFilter
             );
 
+        // TODO: Link to snippet for creating GcpCallInvoker manually from here.
+        /// <summary>
+        /// The maximum number of channels than will be open concurrently to the Bigtable endpoint.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that channels are managed and shared amongst <see cref="BigtableClient"/> instances using the same
```

Comment:
Just to check, that means that the first set of options used to create a client with a given endpoint "wins"? It's not ideal, but I can't see any other way of working that isn't more complex. (I have the same kind of issue in Spanner.)

Reply:
No, the way I have things working in the GcpCallInvokerPool is it will cache based on the pair of endpoint and channel options collection (which is why I manually implemented equality testing for channel options, btw). So if you use a different set of options, which currently just means different values for either of these two properties, you'll get a different GcpCallInvoker.